vm/src/stdlib/array.rs

```diff
@@ -236,6 +236,20 @@ macro_rules! def_array_enum {
                 }
             }
 
+            fn repr(&self, _vm: &VirtualMachine) -> PyResult<String> {
+                // we don't need ReprGuard here
+                let s = match self {
+                    $(ArrayContentType::$n(v) => {
+                        if v.is_empty() {
+                            format!("array('{}')", $c)
+                        } else {
+                            format!("array('{}', [{}])", $c, v.iter().map(|x| x.to_string()).join(", "))
```

Comment:
```suggestion
                            format!("array('{}', [{}])", $c, v.iter().format(", "))
```
[`Itertools::format`](https://docs.rs/itertools/0.9.0/itertools/trait.Itertools.html#method.format)